core/imagedownloader.cpp

```diff
@@ -119,9 +127,11 @@ SHashedImage::SHashedImage(struct picture *picture) : QImage()
 			load(filename);
 			if (!isNull()) {
 				// Make sure the hash still matches the image file
+				qDebug() << "Loaded picture from translated filename " << filename;
```

Comment:
i think the space after `filename` is not needed as each `<<` append is separated by space?

Reply:
applies to similar `qDebug()` calls?